site/pages/docs/ref/cal-events/cal-events-en.hbs

```diff
@@ -29,6 +29,14 @@
 	</ul>
 </section>
 
+<section>
+	<h2>Evaluation and report</h2>
+	<ul>
+		<li>Accessibility (SC WCAG 2.1/2.2 Level A & AA)- <a href="../../../demos/cal-events/reports/a11y-1-en.html">English Report</a></li>
+		<li>Accessibility (SC WCAG 2.1/2.2 Level A & AA)- <a href="../../../demos/cal-events/reports/a11y-1-fr.html">French Report</a></li>
```

Comment:
```suggestion
		<li>Accessibility (SC WCAG 2.1/2.2 Level A &amp; AA)- <a href="../../../demos/cal-events/reports/a11y-1-en.html">English Report</a></li>
		<li>Accessibility (SC WCAG 2.1/2.2 Level A &amp; AA)- <a href="../../../demos/cal-events/reports/a11y-1-fr.html">French Report</a></li>
```

Reply:
Fixed in commit "Updated assessment according to new methodologies"